lib/search/global/search.js

```diff
@@ -39,6 +39,8 @@ module.exports = (client) => {
       }
     };
 
+    query.term = query.term || term;
```

Comment:
Is there an equivalent line needed in the not-global search?

Reply:
I think it's already set here:
https://github.com/UKHomeOffice/asl-search/blob/master/lib/search/establishment-scoped/router.js#L37